README.md

```diff
@@ -12,6 +12,7 @@ An unofficial list of personal and open source projects that are relevant to EA
   - [Open Source Projects](#open-source-projects)
     - [AI Safety](#ai-safety)
     - [General](#general)
```

Comment:
Maybe better to have a General / Multiple Cause areas section?  What do you think?

Reply:
Yeah maybe. I was thinking that maybe [Advice for getting started in Open Source](https://gist.github.com/NicoleJaneway/45069ea3ec808c5507d0e69282976457) should go under an "Advice" category. What do you think of that? I'm okay with either way.

Reply:
Sure!  Not sure how much we'll get in any category.  Let's just put points on the board, and then we can reorganize later.